charts/rasa-x/Chart.lock

```diff
@@ -1,12 +1,12 @@
 dependencies:
 - name: postgresql
   repository: https://charts.bitnami.com/bitnami
-  version: 8.6.4
+  version: 8.6.13
 - name: redis
   repository: https://charts.bitnami.com/bitnami
-  version: 10.5.12
+  version: 10.5.14
 - name: rabbitmq
   repository: https://charts.bitnami.com/bitnami
-  version: 6.18.9
-digest: sha256:29407b1674ec137a7123b46bb549039321efb842e07bf6025ac3a2324e089477
-generated: "2020-03-19T08:55:27.193432+01:00"
+  version: 6.19.2
```

Comment:
did you check what they changed from 6.18 to 6.19? Hopefully didn't rename any `values.yml` keys, did they?

Reply:
I did, they added only two fields, `extraVolumeMounts` and `extraVolumes`. Those two fields us a reason why we bump a rabbitmq chart version.